cypress/e2e/activityHistoryProperty/activityHistoryProperty.js

```diff
@@ -0,0 +1,43 @@
+import { And, Given, Then, When } from "@badeball/cypress-cucumber-preprocessor"
+import ActivityHistoryPropertyPageObjects from '../../pageObjects/activityHistoryPropertyPage'
+
+const activityHistoryProperty = new ActivityHistoryPropertyPageObjects()
+
+Given('I go to the property activity history', () => {
+    cy.getAssetFixture().then((PropertyRecord) => {
+        activityHistoryProperty.visit(PropertyRecord.id)
+        cy.intercept("GET", `*/api/v1/activityhistory*`, { fixture: "activity-history-property.json" }).as("getActivityHistory")
+        cy.wait("@getActivityHistory")
+    })
+})
+
+When('I click close activity history', () => {
+    activityHistoryProperty.closeActivityHistoryButton().click()
+})
+
+Then('the property activity history is displayed', () => {
```

Comment:
We should also do a test where the property activity history displays 'No Activity History' when there are no records in the db

Reply:
Not required as we have unit test to check this in the AH MFE